src/helpers/database/schema.sql

```diff
@@ -17,7 +17,7 @@ CREATE TABLE messages (
   version VARCHAR(6) NOT NULL,
   timestamp BIGINT NOT NULL,
   space VARCHAR(64),
-  type VARCHAR(12) NOT NULL,
+  type VARCHAR(16) NOT NULL,
```

Comment:
Good catch! Actually the db use 24 length for this field, not sure why it showed 12 here.
```suggestion
  type VARCHAR(24) NOT NULL,
```